docs/source/_gallery_original/index.rst

```diff
@@ -3,11 +3,11 @@ AI Gallery
 
 
 AI Gallery is a collection of ready-to-run recipes for popular AI frameworks and AI models.
-It provides a simple way to **package**, **share**, and **distribute** AI projects using the simple interface of SkyPilot.
+It provides a simple way to **package**, **share**, and **run** AI projects using the simple interface of SkyPilot.
 
 Readers can directly execute the SkyPilot YAMLs on their own infrastructure, such as cloud VMs or Kubernetes.
 
-.. image:: https://imgur.com/oHPjeCi.png
+.. image:: https://imgur.com/POFdmiEx.png
```

Comment:
N/A now? https://skypilot.readthedocs.io/en/docs-with-gallery-2/gallery/index.html Shall we use our own?